mysql-test/include/commit.inc

```diff
@@ -620,14 +620,14 @@ create temporary table t2 (a int);
 # One commit for the create temporary table, and two for committing the
 # read of the stored procedure from Aria table (creating temporary table
 # clears the sp cache).
-call p_verify_status_increment(3, 0, 2, 0);
+call p_verify_status_increment(5, 0, 2, 0);
 set sql_mode=default;
 --echo # 19. A function changes temp-trans-table.
 --echo #
 select f1();
 --echo # Two commits because a binary log record is written, and another two
 --echo # as the function f1() is reloaded after creating temporary table.
-call p_verify_status_increment(4, 0, 3, 0);
+call p_verify_status_increment(6, 0, 3, 0);
```

Comment:
likewise, only 4 increments are explained in comments. Same for other result changes.

Reply:
Added.